src/GameKit/GameKit.cs

```diff
@@ -153,8 +153,17 @@ public enum GKTransportType : long {
 		Reliable,
 	}
 
+	[Deprecated (PlatformName.MacOSX, 10,14)]
+	[Deprecated (PlatformName.TvOS, 12,0)]
+	[Deprecated (PlatformName.iOS, 12,0)]
 	[Native]
+#if WATCH && !XAMCORE_4_0
+	// removed in Xcode 10 but a breaking change (for us) to remove
+	[Obsolete ("Not used in watchOS.")]
+#else
+	[Unavailable (PlatformName.WatchOS)]
```

Comment:
Mmh why not `[NoWatch]`?

Reply:
`[NoWatch]` is only usable in binding files, i.e. where the generator process it.
`GameKit.cs` is not processed by the generator so it must be the final (long form) attribute

Reply:
Oh yea I missed that we were not in a binding file.